eng/config/OptProf.json

```diff
@@ -38,10 +38,6 @@
               "filename": "/Microsoft.CodeAnalysis.Remote.Workspaces.dll",
               "testCases":[ "WinForms.OptProfTests.winforms_largeform_vb", "WinForms.OptProfTests.winforms_largeform_cs" ]
             },
-            {
-              "filename": "/Microsoft.CodeAnalysis.EditorFeatures.Wpf.dll",
-              "testCases":[ "WinForms.OptProfTests.winforms_largeform_vb" , "WinForms.OptProfTests.winforms_largeform_cs"]
```

Comment:
FYI to @ryzngard and @phil-allen-msft since this might fix that issue that came by earlier today about IBC.